recipes/libressl/all/conanfile.py

```diff
@@ -52,10 +53,17 @@ def validate(self):
         if self.options.shared and is_msvc(self) and is_msvc_static_runtime(self):
             raise ConanInvalidConfiguration("Static runtime linked into shared LibreSSL not supported")
 
+    def build_requirements(self):
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            # Required for an up-to-date assembler
+            self.tool_requires("binutils/2.41")
```

Comment:
This needs more context, especially since the PR that added a newer version did not require.

In the future, please attach a reason and a build log of the error that required this line for a fix - this is a bit of a red flag without explanation.